worker/src/queues/notify/workers/notifyFailureHandler.ts

```diff
@@ -0,0 +1,70 @@
+import pino from "pino";
+import { Job } from "pg-boss";
+import { NotifyClient, Status, ErrorResponse, GetNotificationsResponse } from "notifications-node-client";
+import config from "config";
+import { NOTIFY_FAILURE_ERRORS } from "./errors";
+
+const queue = "NOTIFY_FAILURE_CHECK";
+const worker = "notifyFailureHandler";
+
+const logger = pino().child({
+  queue,
+  worker,
+});
+
+const notifyClient = new NotifyClient(config.get<string>("Notify.apiKey"));
+
+export async function notifyFailureHandler(job: Job) {
+  const jobId = job.id;
+  logger.info({ jobId }, `received ${worker} job`);
+  try {
```

Comment:
This try-catch is doing more work than it needs to. The longer the try block, the likelihood that there is an error thrown increases, which means you might catch an error that doesn't match up with what you're expecting.

For example, you might get parse errors here, which wouldn't match up with the logged error.